### DISCUSSION TITLE
funktioniert nicht

### DISCUSSION THREAD

**Discussion by Heydo67** [Tampermonkey/tampermonkey#1664] [General]
Trotzt nach Aktualisierung von 15. November 2022 funktioniert immer noch nicht 🤷‍♂️
Bevor Veröffentlichung zuerst testet dem addons bitte😠
Antworten

John WickVor 6 Minuten
instalierte Javascript funtunieren auf Google Chrome nicht.
habe ein bild hochgeladen ( hier  👉https://ibb.co/586rBjs )

mfg...
Löschen

Jan Biniok18.11.2022
Was genau funktioniert nicht? Bitte einen Bug-Report bei Github schreiben. https://github.com/Tampermonkey/tampermonkey/issues

Falls die Javascript-Konsole den Fehler "pagejs missing" zeigt, Tampermonkey deaktivieren, Chrome oder PC neustarten und Tampermonkey wieder aktivieren.

**Comment by derjanb** [comment]
Hi, I need more information.

What is your Chrome and OS version?
What exactly is not working? What should happen, whta happens instead? 
Does the JavaScript console show any errors?
Have you reported issues to the script developers of the not working script?
Does it work with a new temporary profile?
